server/app.js

```diff
@@ -23,11 +26,32 @@ http.listen(3000, () => {
     reportController = new ReportController();
 
     console.log('Listening on port *: 3000');
+    console.log("Creating Tables");
+
+
+    //Create set number of tables on server startup
+
+    for (let x = 1; x <= 10; x++) {
```

Comment:
Should this be hardcoded?  If this is just a temporary value, add a comment above with a TODO tag describing what is left to do, and make a work item in Trello to track it